web/src/scss/utilities/utilities.scss

```diff
@@ -19,6 +19,7 @@
 .u-width--full { width: 100%; }
 .u-width--half { width: 50%; }
 .u-height--half { height: 50% }
+.u-height--97 { height: 97% }
```

Comment:
This feels like something that should be added as a style to a particular component rather than made a utility class. How ofter are we going to be setting elements to 97% height?